### PR TITLE
Refactor consistent code styling

### PR DESCRIPTION
**Pull request by ujjwal-cyph**
Fixes https://app.clickup.com/t/9002019994/CHI-3977
Fixes https://app.clickup.com/t/9002019994/PRF-4662

### CODE REVIEW

File: .github/workflows/ci-tests.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    - name: Restore clang-tools
+      id: cache-clang-tools
+      uses: actions/cache@v3
+      with:
+        path: clang-tools
+        key: clang-tools
+
+    - name: Check formatting
+      env:
+        CACHED: ${{ steps.cache-clang-tools.outputs.cache-hit }}
```
Comment (kartikCypherock) [inline .github/workflows/ci-tests.yml:25]: we should also check in the pre-commit hook in utilities/ci/hooks directory

**Comment by rohanagarwal94** [comment]
Task linked: [CHI-3977 Use a pre-commit hook to format code when merging on github](https://app.clickup.com/t/9002019994/CHI-3977)

**Review by kartikCypherock — APPROVED** [review]

File: .clang-format [hunk 2]
```diff
@@ -1,7 +1,16 @@
 ---
 BasedOnStyle: Chromium
+AlignEscapedNewlines: Right
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortFunctionsOnASingleLine: None
+AllowShortIfStatementsOnASingleLine: Never
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakBeforeMultilineStrings: false
+AlwaysBreakTemplateDeclarations: No
```
Comment (amanCypherock) [inline .clang-format:9]: This is not required as `template` keyword is not present in C.

Comment (amanCypherock) [reply .clang-format:9]:
```suggestion
```

**Review by amanCypherock — APPROVED** [review]

**Pull request merged**